guide/README.md

```diff
@@ -948,6 +967,49 @@ the link below, takes musical paint in a novel direction.
 
 [RUN LIVE](https://walterbender.github.io/musicblocks/index.html?id=1523896294964170&run=True&run=True)
 
+## <a name="ENSEMBLE"></a>3.7 Ensemble
+
+This secction will guide about different Mouse blocks, which returns the data like notes played, current pen color, pitch number etc.
+
+The Mouse index heap block returns a value in the heap at a specified location for a specified mouse.
+
+![alt tag](https://github.com/sugarlabs/musicblocks/blob/master/documentation/turtleheap_block.svg "mouse heap index")
+
+The Mouse sync block aligns the beat count between mice.
+
+![alt tag](https://github.com/sugarlabs/musicblocks/blob/master/documentation/turtlesync_block.svg "mouse sync")
```

Comment:
Do these paths work properly for images? Did you test this?

Reply:
> Do these paths work properly for images? Did you test this?

@walterbender  Yes, I have tested it, it works fine.

Reply:
Doesn't work for me in my tests.

Reply:
@walterbender which one is not working?